content/blog/announcing-hpe-swarm-learning-2-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Announcing HPE Swarm Learning 2.0.0
+date: 2023-05-11T04:47:46.996Z
+featuredBlog: true
+author: HPE Swarm Learning Team
+authorimage: /img/Avatar1.svg
+disable: false
+---
+<!--StartFragment-->
+
+We’re excited to announce HPE Swarm Learning 2.0.0 community release!!
+
+
+In the previous Swarm version, if the sentinel SN goes down during Swarm training, the training process would stop, and there was no way to resume it. However, with this release, we have addressed the issue by implementing a mesh topology(connectivity) between SNs, replacing the previous star topology where only the sentinel SN was connected to other SNs. Also, we now support multiple blockchain miners instead of just one miner in the sentinel SN. Now, even if the initial sentinel SN goes down, since other SNs also function as miners, it allows the training to continue uninterrupted. Additionally, when the initial sentinel SN is down and if a new SN wants to join the network, it can seamlessly integrate and join the Swarm network with the help of any other SN node. This **high availability configuration** ensures improved resilience and robustness of  Swarm Learning.
+
+In Swarm Learning at the sync stage (defined by Sync Frequency), when it is time to share the learning from the individual model, one of the SL nodes is designated as “leader”. This leader node collects the individual models from each peer node and merges them into a single model by combining parameters of all the individuals. **Leader Failure Detection and Recovery (LFDR)** feature enables SL nodes to continue Swarm training during merging process when an SL leader node fails. A new SL leader node is selected to continue the merging process. If the failed SL leader node comes back after the new SL leader node is in action, the failed SL leader node is treated as a normal SL node and contributes its learning to the swarm global model.
```

Comment:
Same remark, please spell out SL the first time you mention it.
...one of the Swarm Learning (SL) nodes is designed as "leader".